packages/sit-onyx/src/styles/mandatory.scss

```diff
@@ -0,0 +1,17 @@
+// `onyx-util-mandatory-marker` is used internally to populate the the "::after" pseudo element with the correct optional or mandatory label
```

Comment:
Imho we should call it "required" instead of "mandatory" to be compliant with the HTML standard

Reply:
sounds good

Reply:
Also the `mandatory.scss` file should be renamed :)

Reply:
done